revocation/crl/fetcher.go

```diff
@@ -51,8 +51,8 @@ type HTTPFetcher struct {
 	// If Cache is nil, no cache is used.
 	Cache Cache
 
-	// DiscardCacheError specifies whether to discard the cache on error.
-	DiscardCacheError bool
+	// DiscardCacheFailure specifies whether to discard any error on cache.
```

Comment:
You probably want to call out that `ErrCacheMiss` is not in the scope.

Reply:
Updated.